release.sh

```diff
@@ -1,16 +1,95 @@
 #!/bin/bash
 
+# Builds and uploads a release, optionally creating a tag for that release.
```

Comment:
Added some more info. This just does the SDK itself (currently just jars, soon also prebuilt versions of `bootstrap`/executor/CLI).